benzene/3d and benzene/3d-keyword-magnet tests failed

For some reason these tests fail after updating my gimic version. I got suspicious and did a complete new clone and the problem remains. Could it be that we need to set a tolerance
delta x for checking because depending on the machines and operating systems the numerics might not be identical? 
On another machine I had the same problem. Therefore I think it is something more serious 
and not related to my local setup. 
Could someone please test and tell if he/she has the same problems? Thank you! 
It is strange that all tests passed on the TravisCI though....

make test
Running tests...
Test project /Users/heike/source/github-gimic/fork/gimic/build
      Start  1: benzene/integration-gauss
 1/22 Test  #1: benzene/integration-gauss .........   Passed    3.79 sec
      Start  2: benzene/integration-lobatto
 2/22 Test  #2: benzene/integration-lobatto .......   Passed    3.82 sec
      Start  3: benzene/vectors
 3/22 Test  #3: benzene/vectors ...................   Passed    1.87 sec
      Start  4: benzene/2d
 4/22 Test  #4: benzene/2d ........................   Passed    1.73 sec
      Start  5: benzene/3d
 5/22 Test  #5: benzene/3d ........................***Failed  551.91 sec
      Start  6: benzene/keyword-magnet
 6/22 Test  #6: benzene/keyword-magnet ............   Passed    3.84 sec
      Start  7: benzene/3d-keyword-magnet
 7/22 Test  #7: benzene/3d-keyword-magnet .........***Failed  579.76 sec
      Start  8: benzene/2d-keyword-magnet
 8/22 Test  #8: benzene/2d-keyword-magnet .........   Passed    2.04 sec
      Start  9: benzene/keyword-spacing
 9/22 Test  #9: benzene/keyword-spacing ...........   Passed   13.19 sec
      Start 10: benzene/keyword-rotation
10/22 Test #10: benzene/keyword-rotation ..........   Passed    2.12 sec
      Start 11: benzene/keyword-rotation_origin
11/22 Test #11: benzene/keyword-rotation_origin ...   Passed    2.39 sec
      Start 12: benzene/keyword-radius
12/22 Test #12: benzene/keyword-radius ............   Passed    2.32 sec
      Start 13: benzene/int-grid-bond-even
13/22 Test #13: benzene/int-grid-bond-even ........   Passed    0.52 sec
      Start 14: benzene/int-cdens
14/22 Test #14: benzene/int-cdens .................   Passed    2.32 sec
      Start 15: benzene/diamag-off
15/22 Test #15: benzene/diamag-off ................   Passed    2.05 sec
      Start 16: benzene/paramag-off
16/22 Test #16: benzene/paramag-off ...............   Passed    2.14 sec
      Start 17: benzene/giao-test
17/22 Test #17: benzene/giao-test .................   Passed    1.34 sec
      Start 18: c4h4/integration
18/22 Test #18: c4h4/integration ..................   Passed    1.03 sec
      Start 19: c4h4/read-grid
19/22 Test #19: c4h4/read-grid ....................   Passed    8.97 sec
      Start 20: open-shell/3d
20/22 Test #20: open-shell/3d .....................   Passed  132.98 sec
      Start 21: open-shell/integration
21/22 Test #21: open-shell/integration ............   Passed    9.63 sec
      Start 22: benzene/skip-jmod-integration
22/22 Test #22: benzene/skip-jmod-integration .....   Passed    2.05 sec

91% tests passed, 2 tests failed out of 22

Total Test time (real) = 1332.02 sec

The following tests FAILED:
	  5 - benzene/3d (Failed)
	  7 - benzene/3d-keyword-magnet (Failed)
Errors while running CTest
make: *** [test] Error 8

--- a/test/benzene/2d-keyword-magnet/test
+++ b/test/benzene/2d-keyword-magnet/test
@@ -12,7 +12,7 @@ assert version_info.major == 2
 options = cli()
 
 # we check entire files
-f = [get_filter(rel_tolerance=1.0e-8)]
+f = [get_filter(rel_tolerance=1.0e-8, skip_below=1.0e-14)]
 
 ierr = run(options,
            configure,

--- a/test/benzene/2d/test
+++ b/test/benzene/2d/test
@@ -12,7 +12,7 @@ assert version_info.major == 2
 options = cli()
 
 # we check entire files
-f = [get_filter(rel_tolerance=1.0e-8)]
+f = [get_filter(rel_tolerance=1.0e-8, skip_below=1.0e-14)]
 
 ierr = run(options,
            configure,

--- a/test/benzene/3d-keyword-magnet/test
+++ b/test/benzene/3d-keyword-magnet/test
@@ -12,7 +12,7 @@ assert version_info.major == 2
 options = cli()
 
 # we check entire files
-f = [get_filter(rel_tolerance=1.0e-8)]
+f = [get_filter(rel_tolerance=1.0e-8, skip_below=1.0e-14)]
 
 ierr = run(options,
            configure,

--- a/test/benzene/3d/test
+++ b/test/benzene/3d/test
@@ -12,7 +12,7 @@ assert version_info.major == 2
 options = cli()
 
 # we check entire files
-f = [get_filter(rel_tolerance=1.0e-8)]
+f = [get_filter(rel_tolerance=1.0e-8, skip_below=1.0e-14)]
 
 ierr = run(options,
            configure,

--- a/test/benzene/vectors/test
+++ b/test/benzene/vectors/test
@@ -11,7 +11,7 @@ assert version_info.major == 2
 
 options = cli()
 
-f = [get_filter(rel_tolerance=1.0e-8)]
+f = [get_filter(rel_tolerance=1.0e-8, skip_below=1.0e-14)]
 
 ierr = run(options,
            configure,

